tests/test_chemfrag.py

```diff
@@ -0,0 +1,1445 @@
+from chemcoord import Cartesian
+from ordered_set import OrderedSet
+from pyscf.gto import M
+
+from quemb.molbe.chemfrag import (
+    ConnectivityData,
+    FragmentedMolecule,
+    SubsetsCleaned,
+    cleanup_if_subset,
+)
+
+
+def test_connectivity_data():
+    m = Cartesian.read_xyz("data/octane.xyz")
+
+    conn_data = ConnectivityData.from_cartesian(m)
+    expected = ConnectivityData(
+        bonds={
```

Comment:
Note that you can use the library without any type-declaration of the integers..

If you enforce type checking, then you have to actually declare the meaning of the integers, i.e.
```python3
bonds = {AtomIdx(0): OrderedSet([AtomIdx(1), ...])
```
this is done in the code, but not in the tests.